static/sass/main.scss

```diff
@@ -0,0 +1,19 @@
+@charset 'UTF-8';
+
+// import vanilla-framework
+@import 'vanilla-framework/scss/build';
+
+// import site specific patterns and overrides
+.asset-thumbnail {
+  max-height: 100px;
+  min-height: 100px;
+}
+
+.navigation {
+  background: #f7f7f7;
```

Comment:
You should use the `$color-navigation-background` variable it set this:
https://docs.vanillaframework.io/patterns/navigation/